### PR TITLE
Home Tab Quick Updates Fix

### PR DESCRIPTION
**Pull request by joeizang**
Fixes an issue where Intro to Solidity points to a different set of tutorials in LearnEth.

### CODE REVIEW

**Review by ryestew — DISMISSED** [review]
One more quick fix - when you click the LearnEth icon on the home tab - it should open the home of LearnEth and not the Remix intro.
image

**Comment by joeizang** [comment]
> One more quick fix - when you click the LearnEth icon on the home tab - it should open the home of LearnEth and not the Remix intro. image

Default seems to be Remix Basics. Tried changing it but it hasn't made a difference.

**Comment by LianaHus** [comment]
Because the method is to open a tutorial. you should probably just start the plugin instead

**Comment by LianaHus** [comment]
it will not work if you already clicked on tutorial before, you need to expose a method

**Comment by joeizang** [comment]
> it will not work if you already clicked on tutorial before, you need to expose a method

This has been addressed.

File: libs/remix-ui/home-tab/src/lib/components/homeTabLearn.tsx [hunk 1]
```diff
@@ -31,13 +31,25 @@ function HomeTabLearn ({plugin}: HomeTabLearnProps) {
...
-  const startLearnEthTutorial = async (tutorial: 'basics' | 'useofweb3js' | 'deploylibraries') => {
+  const startLearnEthTutorial = async (tutorial: 'basics' | 'soliditybeginner' | 'deploylibraries') => {
     await plugin.appManager.activatePlugin(['solidity', 'LearnEth', 'solidityUnitTesting'])
     plugin.verticalIcons.select('LearnEth')
     plugin.call('LearnEth', 'startTutorial', 'ethereum/remix-workshops', 'master', tutorial)
     _paq.push(['trackEvent', 'hometab', 'startLearnEthTutorial', tutorial])
   }
 
+  const goToLearnEthHome = async () => {
+    console.log({ plugin })
```
Comment (Aniket-Engg) [inline libs/remix-ui/home-tab/src/lib/components/homeTabLearn.tsx:42]: remove log

**Review by Aniket-Engg — COMMENTED** [review]
Other things seem fixed. But it still opens the tutorial on clicking on complete bar instead of clicking on `Get Started` button

File: libs/remix-ui/home-tab/src/lib/components/homeTabLearn.tsx (hunk 1 above)
Comment (joeizang) [reply libs/remix-ui/home-tab/src/lib/components/homeTabLearn.tsx:42]: done

**Review by Aniket-Engg — APPROVED** [review]

**Pull request merged**